scripts/get-dep-lib.sh

```diff
@@ -15,7 +15,7 @@ CURRENT_LIBXML2="v2.9.4"
 CURRENT_LIBFTDI="1.3"
 CURRENT_KIRIGAMI="8691651c1f0d09430416ba5fe2130647554b06a9"
 CURRENT_BREEZE_ICONS=""
-CURRENT_GRANTLEE="v5.0.0"
+CURRENT_GRANTLEE="master"
```

Comment:
i don't think we should pin to master, but instead to a stable tag.
v5.0.0 works for us and we shouldn't move away from it if possible.

Reply:
Agree, will remove this.